packages/installer/upgrade.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 set -e
 
-cos upgrade $@
+elemental --debug --logfile /tmp/elemental.log upgrade $@
```

Comment:
Good point I forgot about the logfile in the installer and was about to for the reset command

Reply:
The logfile is also picked up by the test runner if there is a failure and uploaded to github!